modules/virt-sharing-custom-vm-template.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+:_content-type: PROCEDURE
+[id="virt-sharing-custom-vm-template_{context}"]
+= Sharing a custom virtual machine template
+
+Share virtual machine templates to the public {product-title} namespace so that you minimize creating multiple templates in private namespaces.
```

Comment:
This needs to be reviewed by an SME. 
@yfrimanm @ronensdeor Is this statement correct?